### PR TITLE
Update title.txt

### PR DESCRIPTION
**Pull request by gallegonovato**

### CODE REVIEW

File: fastlane/metadata/android/es-ES/title.txt [hunk 1]
```diff
@@ -1 +1 @@
-Administrador de Volumen de Bluetooth
\ No newline at end of file
+Administrador del volumen Bluetooth
```
Comment (d4rken) [inline fastlane/metadata/android/es-ES/title.txt:1]: Title has a limit of 30 characters. Do you have a good idea for a shorter name?

**Review by d4rken — CHANGES_REQUESTED** [review]

File: fastlane/metadata/android/es-ES/title.txt (hunk 1 above)
Comment (gallegonovato) [reply fastlane/metadata/android/es-ES/title.txt:1]: Good afternoon:

Now that you mention it..........

Well, yes ;)

Gestión del volumen Bluetooth
Gestiona el volumen Bluetooth
Maneja el volumen Bluetooth
Administra el volumen Bluetooth
Administrador de volumen Bluetooth

A hug

Comment (d4rken) [reply fastlane/metadata/android/es-ES/title.txt:1]: What is your favorite?

Comment (gallegonovato) [reply fastlane/metadata/android/es-ES/title.txt:1]: Good afternoon @d4rken

Sorry for not telling you which one I would put.

I would put:

Gestión del volumen Bluetooth

Sorry for the mistake of not telling you which one fits best. I don't know where I left my head...

A hug

**Pull request merged**